crates/light-client-prover/src/da_block_handler.rs

```
@@ -139,6 +139,20 @@ where
         last_l1_height_scanned: StartVariant,
         mut shutdown_signal: GracefulShutdown,
     ) {
+        if let Ok(Some(proven_height)) = self
+            .ledger_db
+            .get_highest_l2_height_for_status(sov_db::schema::types::L2HeightStatus::Proven, None)
```

Table is not accessible to lcp. We should propagate the error instead of silently skipping the Err imo.